content/writing_tests/application.md

```diff
@@ -0,0 +1,74 @@
+---
+title : "Application"
+date :  2019-09-12T13:18:58+02:00
+weight : 5
+---
+
+# Interacting With an Application
+
+Now that we have covered how to interact with an operating system and use the screenshot based testing
+capabilities of Sakuli, we can move on how to utilize an one or multiple applications.
+All features are incorporated in the <a href="https://sakuli.io/apidoc/sakuli-legacy/interfaces/thenableapplication.html" target="_blank">ThenableApplication Interface</a>.
+
+
+### Open an application
+
+To open an application, you enter the path to the application or respectively the application name, if you already
+added it to your PATH environment variable. If your path contains spaces, you can escape them with `\\` because parameters
+are separated with spaces.
+{{< highlight typescript >}}
+// path to excel 'C:\\path to excel dir\excel.exe'
+const excel = new Application("C:\\path\\ to\\ excel\\ dir\excel.exe");
+
+// open browser with parameters
+const chromium = new Application("chromium-browser --incognito --proxy-server=localhost:1234");
+
+// path to chrome is e.g. 'C:\\Program\\ Files\google-chrome\chrome.exe'
+const chrome = new Application("C:\\Program\\ Files\google-chrome\chrome.exe --incognito");
+{{< /highlight >}}
```

Comment:
Wouldn't it be necessary to mention, that you have to call `open()` on a new application?